tests/unit/slider/slider_options.js

```diff
@@ -143,6 +143,34 @@ test( "orientation", function( assert ) {
 
 	element.slider( "destroy" );
 
+	element = $( "#slider1" ).slider({
+		range: "max",
+		min: 0,
+		max: 10
+	});
+
+	element.slider( "option", "value", 5 );
+	element.slider( "option", "orientation", "vertical" );
+	equal( element.width(), element.find( ".ui-slider-range " ).width(),
+		"range should fill all horizontal space after changing orientation to vertical" );
```

Comment:
There are explicit values applied in the stylesheet. They're in `em`s though, so you might want to set them to `px` in a `<style>` within the HTML so that you have an exact value to work with.